utils/ccdatofhir/ccda_to_fhir.bal

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// --------------------------------------------------------------------------------------------#
+// Source C-CDA to FHIR - Resource Mappings
+// --------------------------------------------------------------------------------------------#
+
+import ballerinax/health.fhir.r4;
+
+# Transform an C-CDA message to FHIR.
+#
+# + ccda - C-CDA message as a xml
+# + return - FHIR Bundle
+public isolated function ccdaToFhir(xml? ccda) returns r4:Bundle|r4:FHIRError => ccda is xml ? transformToFhir(ccda) :
+    r4:createFHIRError("Failed to parse C-CDA payload to xml", r4:ERROR, r4:INVALID_STRUCTURE);
+
+isolated function transformToFhir(xml xmlDocument) returns r4:Bundle {
+    xmlns "urn:hl7-org:v3" as v3;
```

Comment:
This is already available as a module level variable in mapping_datatypes.bal. Cant we reuse it?

Reply:
Seems that it is not publicly accessible to reuse in other `bal` files. Can not add `public` identifier as well. Is there any other way of doing that?

Reply:
sorry my bad.. thought its the same module.